src/datasources/perf-ds/datasource.js

```diff
@@ -189,6 +189,7 @@ export class OpenNMSDatasource {
         var source = {
           "aggregation": target.aggregation,
           "attribute": target.attribute,
+          "fallback-attribute": target.fallbackAttribute,
```

Comment:
Can we omit this from the query if there is no fallbackAttribute defined? (similar to what we do with the 'subattribute')